packages/flutter_tools/test/commands.shard/hermetic/attach_test.dart

```diff
@@ -285,162 +237,18 @@ void main() {
-      when(device.portForwarder)
-        .thenReturn(portForwarder);
-      when(device.dds)
-        .thenReturn(fakeDds);
-      when(portForwarder.forward(devicePort, hostPort: anyNamed('hostPort')))
-        .thenAnswer((_) async => hostPort);
-      when(portForwarder.forwardedPorts)
-        .thenReturn(<ForwardedPort>[ForwardedPort(hostPort, devicePort)]);
-      when(portForwarder.unforward(any))
-        .thenAnswer((_) async {});
-      when(mockHotRunner.attach(
-        appStartedCompleter: anyNamed('appStartedCompleter'),
-        allowExistingDdsInstance: true,
-        enableDevTools: anyNamed('enableDevTools'),
-      )).thenAnswer((_) async => 0);
-      when(mockHotRunnerFactory.build(
-        any,
-        target: anyNamed('target'),
-        debuggingOptions: anyNamed('debuggingOptions'),
-        packagesFilePath: anyNamed('packagesFilePath'),
-        flutterProject: anyNamed('flutterProject'),
-        ipv6: false,
-      )).thenReturn(mockHotRunner);
-      when(mockHotRunner.exited).thenReturn(false);
-      when(mockHotRunner.isWaitingForObservatory).thenReturn(false);
-
-      testDeviceManager.addDevice(device);
-      when(device.getLogReader(includePastLogs: anyNamed('includePastLogs')))
-        .thenAnswer((_) {
-          // Now that the reader is used, start writing messages to it.
-          fakeLogReader.addLine('Foo');
-          fakeLogReader.addLine(
-              'Observatory listening on http://127.0.0.1:$devicePort');
-          return fakeLogReader;
-        });
-      final File foo = globals.fs.file('lib/foo.dart')
-        ..createSync();
-
-      // Delete the main.dart file to be sure that attach works without it.
-      globals.fs.file(globals.fs.path.join('lib', 'main.dart')).deleteSync();
-
-      final AttachCommand command = AttachCommand(hotRunnerFactory: mockHotRunnerFactory);
-      await createTestCommandRunner(command).run(<String>[
-        'attach',
-        '-t',
-        foo.path,
-        '-v',
-        '--device-user',
-        '10',
-        '--device-timeout',
-        '15',
-      ]);
-      final VerificationResult verificationResult = verify(
-        mockHotRunnerFactory.build(
-          captureAny,
-          target: foo.path,
-          debuggingOptions: anyNamed('debuggingOptions'),
-          packagesFilePath: anyNamed('packagesFilePath'),
-          flutterProject: anyNamed('flutterProject'),
-          ipv6: false,
-        ),
-      )..called(1);
-
-      final List<FlutterDevice> flutterDevices = verificationResult.captured.first as List<FlutterDevice>;
-      expect(flutterDevices, hasLength(1));
-      final FlutterDevice flutterDevice = flutterDevices.first;
-      expect(flutterDevice.userIdentifier, '10');
-    }, overrides: <Type, Generator>{
-      FileSystem: () => testFileSystem,
-      ProcessManager: () => FakeProcessManager.any(),
-    });
-
-    testUsingContext('fallbacks to protocol observatory if MDNS failed on iOS', () async {
```

Comment:
I removed this test because It was actually trying to spin up an mdns connection and then crashing on a windows host. We have separate fallback tests anyway